src/sass/Fabric.Typography.Output.scss

```diff
@@ -198,7 +198,7 @@
 
 .ms-fontColor-neutralSecondary, 
 .ms-fontColor-neutralSecondary--hover:hover {
- @include ms-fontColor-neutralSecondary;
+  @include ms-fontColor-neutralSecondary;
```

Comment:
It was only one space instead of two.  This fixes it to be consistent with the rest of the include rules.